pkg/arvo/sur/post.hoon

```diff
@@ -0,0 +1,37 @@
+/-  *resource
+|%
++$  atom        @u
```

Comment:
`$atom` is a system type referring to `@`, so let's not redefine it here.  I'm not sure there's any material advantage of using the `@u` aura here.  I'd be inclined to just remove this type alias.

Reply:
Alternatively, if this is used as a sequence number, then calling it that could be useful, as well as adding a more specific aura like `@udsequence`.

Reply:
Is there an alternative named type for "@" that I can reference, akin to `term` for @tas?

Reply:
`atom` is `@` in hoon.hoon, is that what you're asking?

Reply:
Thanks!